internal/usage/usage.go

```diff
@@ -229,7 +229,7 @@ const (
 	ArchiveAfter                             = "Number of days that specifies the age limit for the data in the live Atlas cluster."
 	TargetProjectID                          = "Unique identifier of the project that contains the destination cluster for the restore job."
 	APIAccessListIPEntry                     = "IP address to be allowed for a given API key."
-	NetworkAccessListIPEntry                 = "IP address to be allowed to access the deployment."
+	NetworkAccessListIPEntry                 = "IP address to be allowed to access the deployment. Set to 0.0.0.0/0 if you want to enable connection from anywhere; use comma(,) to separate multiple entries."
```

Comment:
@dianchenghu FYI it be better for users to add `--allowAll` flag to the commands (name open to suggestions)

Reply:
not sure I follow, do you mean we should remove the comma suggestion?

Reply:
we should remove the whole `Set to 0.0.0.0/0 if you want to enable connection from anywhere`

also the comma is not totally correct for flags, repeating the flag is also correct are more common`--accessList <entry1> --accessList <entry2>` and why we stick to singular names on our flags

Reply:
I'm OK w/ adding the comment about the commas in another variable and keeping this `NetworkAccessListIPEntry` as it was before. I'm not sure it is a good mentioning flag within a command description, look redundant? 

Wdyt?

Reply:
actually, this usage is only for the quickstart, so it should be OK to update it, I can remove the extension for now and follow up with Diancheng in the right wording?

Reply:
I wish this discussion was during the design phase.  The reason why we are adding the copy is during user testing someone asked how can I allow from anywhere. Since we have the -allowAll flag, I think we can mention it in the help text.   So suggestion for the help text is:

IP address to be allowed to access the deployment. Use comma(,) to separate multiple entries. Use "--allowAll" to enable connection from anywhere.

FYI  @boooczek

Reply:
I updated with the phrase proposed by Diancheng and kept the flag untouched. LMK if that works for you both.